src/utils/mapsheet.ts

```diff
@@ -89,7 +89,7 @@ export const MapSheet = {
   gridSizeMax: 50000,
   roundCorrection: 0.01,
   /** Allowed grid sizes, these should exist in the LINZ Data service (meters) */
-  gridSizes: [10_000, 5_000, 2_000, 1_000, 500],
```

Comment:
When fixing bugs it would be nice to add a unit test.

Reply:
It wasn't clear what the best way to do this was, I think the tests might need to be reviewed.

Reply:
while this might hide the error, I don't think this fixes the bug, the tiles will not be named correctly when using 1:50k.

Reply:
like this? https://toitutewhenua.atlassian.net/browse/TDE-1014